docs/release-notes/8.13.asciidoc

```diff
@@ -0,0 +1,49 @@
+* You can isolate and release a SentinelOne-protected host from detection alerts and the response console, and view third-party actions in the response actions history log ({pull}173927[#173927], {pull}175810[#175810]).
+* Allows you to enable and disable cloud security Benchmark rules ({pull}174575[#174575]).
+
+
+[discrete]
+[[enhancements-8.13.0]]
+==== Enhancements
+
+* Enables advanced sorting and customization options for the the Findings page's **Vulnerabilities** table ({pull}174413[#174413]).
+* Adds the ability to analyze an event within a specific time range and data view ({pull}176364[#176364]).
+* Enables the newly expanded host and user details flyouts, which allow you to view host or user details, risk data and inputs, and asset criticality ({pull}175899[#175899]).
+* Improves the header layout in the alert details flyout so basic alert details are better organized ({pull}175075[#175075]).
+* Adds inline actions and a search bar to the left panel in the event analyzer UI and improves formatting issues ({pull}172397[#172397]).
+
+[discrete]
+[[bug-fixes-8.13.0]]
+==== Bug fixes
+
+* Fixes a bug in Timeline that prevented events from being unpinned after their associated notes were deleted ({pull}178537[#178537]).
+* Fixes a bug that prevented the event analyzer preview from loading properly for {esql} rules ({pull}178389[#178389]).
+* Fixes a bug that prevents you from editing, adding, or removing query filters when creating or editing a custom query, indicator match, or new terms rule ({pull}178207[#178207]).
+* Fixes a bug that caused unnecessary error messages to appear on the MITRE ATT&CK Coverage Overview dashboard ({pull}178126[#178126]).
+* Prevents an infinite loading state on the Add Rules page for users with limited permissions ({pull}178005[#178005]).
+* Fixes a bug that prevented the **Reset Fields** action on the Alerts table from resetting the table's columns ({pull}177986[#177986]).
+* Fixes a bug that interfered with the rule filtering interface when you opened it from specific parts of {elastic-sec} ({pull}177946[#177946]).
+* Ensures that text within the risk score preview table translates correctly ({pull}177680[#177680]).
+* Fixes a bug that could prevent the correct `kibana.alert.threshold_result.terms.value` field value from appearing in the alert details flyout ({pull}177472[#177472]).
+* Fixes multiple bugs affecting the rule filters UI in the Create rule interface ({pull}177081[#177081]).
+* Updates the alert assignment UI to make its data model and intended usage clearer ({pull}176442[#176442]).
+* Fixes rule overwrite behavior when importing new rules. Now, when a new rule overwrites an existing rule, the old rule's fields are not included in the new rule, even when the two rules have different fields ({pull}176166[#176166]).
```

Comment:
Nit: this would be a bit easier to understand for my technical brain, but I'm fine to keep it as is too:

```suggestion
* Fixes rule overwrite behavior when importing new rules. Now, when a new rule overwrites an existing rule, the new rule completely replaces all the fields of the old one, and the old rule's fields are never included in the new rule ({pull}176166[#176166]).
```